web-external/src/components/process-data.jsx

```diff
@@ -2,7 +2,9 @@
 
 import React from 'react';
 
+import { partial } from 'underscore';
```

Comment:
The switch is still stuck in #29, but I wanted to make sure you knew that we're switching to lodash.